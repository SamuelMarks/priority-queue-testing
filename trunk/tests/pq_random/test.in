comm This is a test file
init 2
reps 3
with ins
with dcr
with dmn
seed 23928103
prio 100
