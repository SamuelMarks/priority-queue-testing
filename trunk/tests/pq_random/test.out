com  This is a test file
com ## Init=2, Reps=3 
com ## With= ins dcr dmn 
com ## Max Priority= 100 (+epsilon)
com ## Seed: 23928103
ins 51.500000 1
ins 96.333333 2
ins 58.250000 3
dcr 55.250000 3
dmn
ins 5.200000 4
dcr 43.250000 3
dmn
ins 82.166667 5
dcr 61.166667 5
dmn
